bad_packages.md

```diff
@@ -0,0 +1,43 @@
+A few errors found by randomly installing some lwt packages.
+
+All seem to point to libraries using lwt.unix but only linking to lwt.
```

Comment:
Agreed, thanks for checking this. I don't think this blocks the PR. We should remove this file from the branch; I've copied it out locally (or we can save it in a gist). I'll do builds of reverse dependencies and notify the package maintainers after the PR.

I'll give some thought to whether this means we have to do this conversion softer until 4.0.0. But, regardless, if we really have to, we can, I think, make this transition smoother by adding some symlinks after install in 3.1.0, or moving installed files and patching `META`, ugly as that is. In any case, we can decide that after merge.